app/models/order.rb

```diff
@@ -12,6 +12,6 @@ def formatted_created_at
   end
 
   def total_dollar_amount
-    number_to_currency(total_price / 100)
+    number_to_currency(total_price / 100.00)
```

Comment:
Yes, I think there are other places that we need to do that as well (we really should pull all of them out into a helper method).